.travis.yml

```diff
@@ -33,6 +33,13 @@ env:
 matrix:
   fast_finish: true
   include:
+    - env:
+      - HADOLINT="$HOME/hadolint"
+      install:
+      - sudo apt-get install jq
```

Comment:
Use

```yml
apt:
  packages:
    - jq
```

Reply:
Actually scratch that. jq is part of the standard Travis image.

Reply:
I quickly looked at this job https://travis-ci.org/FreshRSS/FreshRSS/jobs/562212239 and for me it looks like that jq isn't installed by default.

Reply:
Line 55 of the log. ;-)

![Screenshot_2019-07-23_07-47-34](https://user-images.githubusercontent.com/202757/61685841-2c938400-ad1e-11e9-90f1-5ff22291d4d0.png)

Reply:
🤔 seems legit but why did I install it then?